### DISCUSSION TITLE
Host a Powertools office hours session

### DISCUSSION THREAD

**Discussion by michaelbrewer** [aws-powertools/powertools-lambda-python#343] [General]
I think it would be great to have an office hours session on Twitch or Youtube, where contributors and customers of the Powertools can bring up questions and we can talk about new ideas and roadmap.

**Comment by mwarkentin** [comment]
There was a good article about this recently: https://simonwillison.net/2021/Feb/19/office-hours/

**Reply by heitorlessa** [reply]
Update - @am29d @t1agob and I are discussing this now.

**Reply by michaelbrewer** [reply]
@heitorlessa any updates on this? Things Powertools was gaining a lot of support at AWS (VPs and CTO tweeting about it). We just keep the momentum going!
